jenkins-agent.ps1

```diff
@@ -1,6 +1,6 @@
 # The MIT License
 #
-#  Copyright (c) 2019, Alex Earl
+#  Copyright (c) 2020, Alex Earl
```

Comment:
Usually copyright dates include the first year the work was created and at least the most recent year of modification.  Dropping the year of creation might be viewed as an attempt to extend the length of the copyright beyond copyright allowed limits.
```suggestion
#  Copyright (c) 2019-2020, Alex Earl
```

Reply:
Good catch. I was typing too quickly.